pithos/plugins/notification_icon.py

```diff
@@ -112,6 +79,18 @@ def check_for_plasma():
                 None
             )
 
+        def on_StatusNotifierWatcher_finish(source, result, data):
+            try:
+                StatusNotifierWatcher = Gio.DBusProxy.new_finish(result)
+            except GLib.Error as e:
+                logging.debug(e)
+                check_for_x()
+            else:
+                if StatusNotifierWatcher.get_name_owner():
+                    init_appindicator()
+                else:
+                    check_for_plasma()
```

Comment:
We still need ot check for Plasma though if AppIndicator3 is not installed though. It's possible to have AppIndicator not installed, and that bus still owned on Plasma. Plasma doesn't support Gtk.StatusIcon.